lib/src/debian.dart

```diff
@@ -0,0 +1,216 @@
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:cli_pkg/src/standalone.dart';
+import 'package:crypto/crypto.dart';
+import 'package:grinder/grinder.dart';
+import 'package:path/path.dart' as p;
+import 'package:pub_semver/pub_semver.dart';
+
+import 'config_variable.dart';
+import 'github.dart';
+import 'info.dart';
+import 'utils.dart';
+
+/// The GitHub repository slug (for example, `username/repo`) of the PPA
+/// repository for this package.
+///
+/// This must be set explicitly.
+final debianRepo = InternalConfigVariable.fn<String>(
+    () => fail("pkg.debianRepo must be set to deploy to PPA repository."));
+
+/// The email address for signing the package.
+final signingEmail = InternalConfigVariable.fn<String>(
```

Comment:
I suggest using a different name than `email` for the config variable of the GPG key id. The recommended way to reference a private key is to use the fingerprint rather than the email (and the import-gpg action for GHA is providing the fingerprint, so that would be what gets used in the sass automation)

Reply:
Yes, that'd be better. I'll update